### ISSUE TITLE
[Bug]: Screen reader is announcing incorrect row and columns count using table shortcut key 'T' in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#10414]
### Link to reproducible scenario

https://www.ag-grid.com/react-data-grid/grouping/

### Describe the bug

Steps to reproduce 

1. Open [React Grid: Row Grouping | AG Grid](https://www.ag-grid.com/react-data-grid/grouping/) in edge browser. 
2. AG Grid page will open.
3. Press tab key to reach country control.
4. Now press table shortcut key 'T'.
5. Verify using table shortcut key 'T' screen reader is announcing incorrect row and columns count or not.

Actual behaviour
Screen reader is announcing incorrect row and columns count using table shortcut key 'T' in Windows.

https://github.com/user-attachments/assets/2da3b633-b677-4e53-afbd-89330d35a937

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/eac502c9-43b6-4af6-8694-3308ead1302a" />

Expected behaviour
Screen reader should announce correct row and columns count using table shortcut key 'T' in Windows.
It should announce as a '512 rows and 6 columns row 1 column 1 clickable sorted group'.

More information
Title: Screen reader is announcing incorrect row and columns count using table shortcut key 'T' in Windows.
User Impact: Users who depend on screen readers for navigation may find it difficult to understand the structure of the table. Incorrect row and column counts can lead to confusion and frustration as they try to determine the correct layout and content of the table.

### Version

30.1.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by mmorris8** [comment]
The aria tags are in an invalid state - that may be the reason for this. See https://github.com/ag-grid/ag-grid/issues/11363

**Comment by AG-Zoheil** [comment]
Hi

Thank you for reporting this.

The grid version this is reported in is `v30.`

The grid has naturally advanced a lot since then. If you are still having problems on our latest version, can we please ask you to open a new issue with a reproducible live scenario - you can use one of the plunker examples from our [documentation](https://ag-grid.com/javascript-data-grid/) as a starting point.

This would help our team support in the most efficient way, and also increase visibility for the wider AG Grid community in case of contributions and comments.

Much appreciated!

Kind regards,
Zoheil
